plugins/quickinspector/quickoverlay.cpp

```diff
@@ -437,17 +441,18 @@ void QuickOverlay::windowAfterRendering()
         m_grabbedFrame.transform.translate(intersect.x() * m_renderInfo.dpr , -intersect.y() * m_renderInfo.dpr - h);
         m_grabbedFrame.image.setDevicePixelRatio(m_renderInfo.dpr);
 
-        if (!m_grabbedFrame.image.isNull()) {
-            m_sceneGrabbed.invoke(this, Qt::QueuedConnection, Q_ARG(GammaRay::GrabbedFrame, m_grabbedFrame));
-        }
+        // Let emit the signal even if our image is possibly null, this way we make perfect ping/pong
+        // reuests making it easier to unit test.
```

Comment:
Easier to unit test is not the design criteria here ;-) Can the client handle empty frames? How often does  this happen, ie. is this causing unreasonable performance costs?